documentation/airport-guides/kbos.md

```diff
@@ -0,0 +1,66 @@
+### About this facility
+Boston TRACON (A90), Merrimack, New Hampshire, USA
+658,524 annual operations in 2017 ([FAA ATADS](https://aspm.faa.gov/opsnet/sys/Tracon.asp))
+
+Located almost 40 miles Northwest of KBOS, this consolidated approach control facility is one of six large TRACON's in the United States. It controls a large portion of the states of Massachusetts and New Hampshire
```

Comment:
TRACONs, assuming you want the plural
's is possessive (ie. Kim's car)

Full stop/period (whatever you call it) after New Hampshire